src/shogun/evaluation/TimeSeriesSplitting.h

```diff
@@ -40,55 +40,60 @@
 namespace shogun
 {
 	class CLabels;
-
-	/** @brief Implements a timeseries splitting strategy for cross-validation.
-	 * The strategy builds a given number of subsets each filled with labels
-	 * indices greater than a split index. The split indices are \f$ c[N/K] \f$
+	/** @brief Implements a timeseries splitting strategy for cross-validation,
+	 * respecting time.
+	 * Each fold splits timeseries into train (subset_inverse) and validation
```

Comment:
pls remove the (subset_inverse) here. This is documented in the base class
The newlines are awkwards, could you remove them?